README.md

```diff
@@ -89,7 +89,10 @@ limited by the capacity of a single physical machine, lest we sacrifice the
 property of complete auditability.  The Lightning Network [\[7\]][7] can help
 scale Bitcoin transaction volume by leaving some transactions off the ledger
 completely and is well suited for micropayments and privacy preserving payment
-rails, but may not be suitable for more generalized scaling needs.
+rails, and it is suitable for more generalized scaling needs.
+<!---
+state-channels allow for generalized smart contracts in the lightning network
+--->
```

Comment:
yes.

You don't need channels to fix this bug. Channels are how I happened to solve it for flying fox.